### DISCUSSION TITLE
trail upload to cycling.waymarkedtrails

### DISCUSSION THREAD

**Discussion by whzahp** [waymarkedtrails/waymarked-trails-site#406] [Q&A]
Hi,

I have uploaded the below gpx file in OSM with the tags I believe are correct.
https://www.openstreetmap.org/user/whzahp/traces/4017195 

However, the above route does not seem to reflect in https://cycling.waymarkedtrails.org/ even after 2 days.
Could you kindly let me know what I can do next?

Thank you,
Rishi

**Comment by lonvia** [comment]
I suspect you misunderstand what OSM is about. OSM collects geographic information that is verifiable on the ground. In the case of cycling routes, these are (mostly officially) established routes that are marked on the ground through signs, information boards etc. It is not a collection of personal routes. There are apps like Komoot for that.

That said, if the route you have tried to upload is an officially established route, then Supplying the GPX file is not enough. You need to 'map' the route as a relation in the OSM database. LearnOSM has a [short section about mapping routes](https://learnosm.org/en/josm/josm-relations/). You might also want to look through the other parts of this guide to get an idea how to contribute to OSM.

**Reply by whzahp** [reply]
Thank you for the understanding. 
There are only few discrete sections of dedicated cycling tracks in the city I live.
I think OSM cycling is not very applicable then. 
I will checkout Komoot for sharing my personal route.
Thank you again for the clarification.
